src/Components/WebView/WebView/src/WebViewManager.cs

```diff
@@ -210,7 +212,9 @@ protected virtual void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-                    _currentPageContext?.Dispose();
+                    // This is the same pattern that is used in Microsoft.Extensions.Hosting.Internal.Host.Dispose():
+                    //    https://github.com/dotnet/runtime/blob/645dfe69fb25407de7a5484d7a03d3342b4e0279/src/libraries/Microsoft.Extensions.Hosting/src/Internal/Host.cs#L146
+                    _currentPageContext?.DisposeAsync().AsTask().GetAwaiter().GetResult();
```

Comment:
Can WebViewManager implement DisposeAsync?

Reply:
Good idea, I can push this down 1 more layer (where maybe in theory some callers actually support async disposal).